compiler/damlc/BUILD.bazel

```diff
@@ -101,6 +101,7 @@ da_haskell_library(
         "haskell-lsp",
         "lens-aeson",
         "lens",
+        "listsafe",
```

Comment:
I assume the function you take from this library is already contained in the `safe` package.

Reply:
Apparently not : Without `listsafe`, get `Could not find module ‘Data.List.Safe’`.

Reply:
The function is called [lastMay](https://hackage.haskell.org/package/safe-0.3.17/docs/Safe.html#v:lastMay) and it’s in the `Safe` module.

Reply:
Ok.